CONTRIBUTING.md

```diff
@@ -6,13 +6,13 @@ possible.
 ... (in particular how this is synced with internal changes to the project)
 
 ## Pull Requests
-We actively welcome your pull requests.
```

Comment:
We just need a newline here. We don't need to do anything fancy with trailing whitespace.